.github/workflows/cli-jobs-pipelines-cifar-10-pipeline.yml

```diff
@@ -5,6 +5,7 @@ on:
     - cron: "0 0/4 * * *"
   pull_request:
     branches:
+      - march-cli-preview
```

Comment:
should we keep them here?

Reply:
As last discussion, we plan to add gate for this branch?